doc/user/ruby-managers.md

```diff
@@ -1,8 +1,40 @@
-# Ensuring Ruby managers are configured correctly
+# Configuring Ruby managers
 
-If you are using a Ruby manager like `rvm`, `rbnev`, or `chruby` make 
-sure that the manager does not set environment variables `GEM_HOME`, `GEM_PATH`,
-and `GEM_ROOT`. The variables are picked up by truffleruby (as any other Ruby 
+It's recommended to add TruffleRuby to a Ruby manager for ease of use.
+
+## RVM
+
+RVM has a command for adding precompiled Ruby.
```

Comment:
RVM has a command for adding a precompiled Ruby to the list of available rubies: